canonical/vanilla-framework
# Use new theme colours in base styles

## Done

Updates some remaining Vanilla base styles to use new themed colours.

Fixes #5025 
Fixes [WD-10224](https://warthogs.atlassian.net/browse/WD-10224)

## QA

- Open [demo](https://vanilla-framework-5054.demos.haus/)
- Review examples, make sure they look fine in different themes
- Slider:
  - https://vanilla-framework-5054.demos.haus/docs/examples/base/forms/range
  - Check both in Chrome and Firefox
- Forms
  - https://vanilla-framework-5054.demos.haus/docs/examples/base/forms/labels
- Blockquote
  - https://vanilla-framework-5054.demos.haus/docs/examples/base/blockquotes
  
### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)



[WD-10224]: https://warthogs.atlassian.net/browse/WD-10224?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
